source/apiVolontaria/apiVolontaria/views.py

```diff
@@ -121,15 +122,15 @@ def get_authenticators(self):
     def post(self, request, *args, **kwargs):
         response = self.create(request, *args, **kwargs)
         # get token for user save and send key this token with a mail
-        user = User.objects.get(username=request.data["username"])
+        user = User.objects.get(username = request.data["username"])
```

Comment:
no space around `=` for parameters (PEP8)

Reply:
done